plugins/modules/cloud/opennebula/one_vm.py

```diff
@@ -80,7 +80,8 @@
       - C(poweredoff) - power-off instances
       - C(rebooted) - reboot instances
       - C(absent) - terminate instances
-    choices: ["present", "absent", "running", "rebooted", "poweredoff"]
+      - C(release) - releases instances created with C(vm_start_on_hold). This has been added in community.general 4.4.0.
```

Comment:
```suggestion
      - C(released) - releases instances created with C(vm_start_on_hold). This has been added in community.general 4.4.0.
```
`state` is a state, not an operation. `release` is an operation, `released` a state.

Reply:
please mention the new state in the `description` for `instance_ids` above too (GitHub won't let me comment there, as it's out of diff range...)